fastsim-core/src/vehicle/powertrain/prepare_eff_values.py

```diff
@@ -0,0 +1,439 @@
+#%%
```

Comment:
What's `#%%` doing in here?  I think it'd be better to delete that.  

@robinsteuteville Could you put a doc string at the top of this file?  The very first thing in the file should be
```python
"""
this is is a single-line or multi-line doc string explaining why this file exists, what it does, and how to use it
"""

Reply:
@robinsteuteville , also I'm pretty baffled by a python file living under `fastsim-core/src/`

Reply:
@calbaker see my earlier teams comment: "I also added a prepare_eff_values.py file to the same folder (wasn't sure where to add it), since I wanted to document how to transition the Altrios arrays to fastsim -- they needed to be transposed in a specific way to work given how the arrays in the interpolator need to be structured. That file can be moved or gotten rid of if desired."
I added this here more as a temporary measure, since I thought it was something that might need to be documented and I wasn't sure where it needed to be stored.

Reply:
@robinsteuteville, I'll need to give this some thought.